src/frontend/src/elements/YearInputHandler/hooks/useSelectedYears.ts

```diff
@@ -0,0 +1,40 @@
+import { useSearchParams } from "react-router-dom";
+
+type useSelectedYearsReturnType = {
+    selectedYears: number[],
+    addSelectedYear: (year: number) => void,
+    removeSelectedYear: (year: number) => void,
```

Comment:
kannst du setYears auch noch hinzu packen? (auch wenn es nicht verwendet wird)

Reply:
Ich habe es eigentlich explizit nicht hinzugefügt weil ich mir dachte:
Der Nutzer hat in der UI nie eine option mehrere Years hinzuzufügen oder mehrere Years zu entfernen. Und das ist auch gut so, da wir dadurch eine viel simplere UI/UX bieten. 

Mit dem Type haben wir auch eine schöne logische Übersicht, wie der Nutzer den State der SPA verändern kann.